rakefile.rb

```diff
@@ -1,197 +1,29 @@
-require 'open-uri'
```

Comment:
seems the rakefile.rb was copied from ospsuite-R repo. Things like installing PK-Sim, creating special linux builds etc. are not required here, only the version replacement in the DESCRIPTION file before build.
So in fact this rakefile.rb is copied from the tlf repo :)

Reply:
Thanks for the cleanup! Don't know Ruby, so didn't want to break something unintentionally :)